Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/UpdateParameter.cs

```diff
@@ -3,44 +3,50 @@
 using Objects.Geometry;
 using System.Collections.Generic;
 using System.Linq;
+using Speckle.Core.Helpers;
+using Speckle.Core.Models;
 
 
 namespace Objects.Converter.Revit
 {
   public partial class ConverterRevit
   {
-    public void UpdateParameter(ParameterUpdater paramUpdater)
+    public ApplicationObject UpdateParameter(ParameterUpdater paramUpdater)
     {
       //the below does not work because ApplicationIds are stored within a stream
       //try to get element using ApplicationId
       //this will only work if the element has been successfully been received in Revit before
       //var element = GetExistingElementByApplicationId(paramUpdater.revitId);
-
+      
+      var appObj = new ApplicationObject(paramUpdater.id, paramUpdater.speckle_type) { applicationId = paramUpdater.applicationId };
       Element element = null;
 
       //try to get element using ElementId
-      int intId;
-
-      if (int.TryParse(paramUpdater.elementId, out intId))
+      if (int.TryParse(paramUpdater.elementId, out int intId))
       {
         var elemId = new ElementId(intId);
         element = Doc.GetElement(elemId);
       }
 
       //try to get element using UniqueId
-      if (element == null)
+      element ??= Doc.GetElement(paramUpdater.elementId);
+
+      if (element != null)
       {
-        element = Doc.GetElement(paramUpdater.elementId);
+        SetInstanceParameters(element, paramUpdater);
+        appObj.Update(status: ApplicationObject.State.Updated, logItem: $"Successfully updated instance parameters for element {paramUpdater.elementId}");
```

Comment:
where is this logItem going? I'm a bit meh about "I did this thing and it worked" messages as it bloats logging, happy to discuss

Reply:
this gets added to the list of logs for that "application object", and displayed to the user in the report tab.

![Screenshot 2023-11-02 at 11 34 50](https://github.com/specklesystems/speckle-sharp/assets/2316535/bfd64487-e3da-4dbd-8ba0-2f4b9119aee4)

Our converter/conversion logging is not great, but at least this aligns it with everything else. Not sure how good/bad that is 😅 Happy to discuss this a bit further in a call if needed!

Reply:
If this is current practise then let's continue as is, but it feels like logging policy is needed